tests/server/schemas/test_schedules.py

```diff
@@ -353,144 +419,228 @@ async def test_interval_schedule_hourly_daylight_savings_time_forward_with_UTC(
         """
         On 3/11/2018, at 2am, America/New_York switched clocks forward an hour.
         """
-        dt = datetime(2018, 3, 10, 23, tz="America/New_York")
+        dt = datetime(2018, 3, 10, 23, tzinfo=ZoneInfo("America/New_York"))
         s = IntervalSchedule(interval=timedelta(hours=1))
         dates = await s.get_dates(n=5, start=dt)
         # skip 2am
-        assert [d.in_tz("America/New_York").hour for d in dates] == [23, 0, 1, 3, 4]
+        assert [d.astimezone(ZoneInfo("America/New_York")).hour for d in dates] == [
+            23,
+            0,
+            1,
+            3,
+            4,
+        ]
         # constant hourly clock in utc time
-        assert [d.in_tz("UTC").hour for d in dates] == [4, 5, 6, 7, 8]
+        assert [d.astimezone(ZoneInfo("UTC")).hour for d in dates] == [4, 5, 6, 7, 8]
 
     async def test_interval_schedule_hourly_daylight_savings_time_forward(self):
         """
         On 3/11/2018, at 2am, America/New_York switched clocks forward an hour.
         """
-        dt = datetime(2018, 3, 10, 23, tz="America/New_York")
+        dt = datetime(2018, 3, 10, 23, tzinfo=ZoneInfo("America/New_York"))
         s = IntervalSchedule(interval=timedelta(hours=1), timezone="America/New_York")
         dates = await s.get_dates(n=5, start=dt)
         # skip 2am
-        assert [d.in_tz("America/New_York").hour for d in dates] == [23, 0, 1, 3, 4]
+        if sys.version_info >= (3, 13):
+            assert [d.hour for d in dates] == [
+                23,
+                0,
+                1,
+                3,
+                4,
+            ]
         # constant hourly clock in utc time
-        assert [d.in_tz("UTC").hour for d in dates] == [4, 5, 6, 7, 8]
+        assert [d.astimezone(ZoneInfo("UTC")).hour for d in dates] == [4, 5, 6, 7, 8]
 
     async def test_interval_schedule_hourly_daylight_savings_time_backward(self):
         """
         11/4/2018, at 2am, America/New_York switched clocks back an hour.
         """
-        dt = datetime(2018, 11, 3, 23, tz="America/New_York")
+        dt = datetime(2018, 11, 3, 23, tzinfo=ZoneInfo("America/New_York"))
         s = IntervalSchedule(interval=timedelta(hours=1), timezone="America/New_York")
         dates = await s.get_dates(n=5, start=dt)
 
-        assert [d.in_tz("America/New_York").hour for d in dates] == [23, 0, 1, 2, 3]
-        # skips an hour UTC - note interval clocks skip the "6"
-        assert [d.in_tz("UTC").hour for d in dates] == [3, 4, 5, 7, 8]
+        if sys.version_info >= (3, 13):
+            # Hour is repeated because the interval is 1 hour
+            assert [d.astimezone(ZoneInfo("America/New_York")).hour for d in dates] == [
+                23,
+                0,
+                1,
+                1,
+                2,
+            ]
```

Comment:
This assertion is different for Python 3.13 because `whenever` handles DST differently and arguably more correctly as it matches the behavior written in our docs: https://docs.prefect.io/v3/automate/add-schedules#interval